src/core/components/AMInstallButton/index.js

```diff
@@ -91,12 +99,28 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
     puffy: true,
   };
 
+  trackInstallClick() {
+    const { _log, _tracking, experimentEnabled, variant } = this.props;
+
+    if (!experimentEnabled) {
+      _log.debug(`Experiment "${EXPERIMENT_ID}" is disabled.`);
+      return;
+    }
+
+    _tracking.sendEvent({
+      action: variant,
+      category: EXPERIMENT_CATEGORY,
+    });
+  }
+
   installTheme = (event: SyntheticEvent<HTMLAnchorElement>) => {
     const { addon, status, installTheme } = this.props;
 
     event.preventDefault();
     event.stopPropagation();
 
+    this.trackInstallClick();
```

Comment:
We already track the install and activation events so I don't think you need to duplicate the click event here. I'm not an analytics pro but I think all we need to do is add one event (on `componentDidMount`, maybe?) to link the user to the variant of the button. After that we should be able to visualize conversions but also see which conversion is linked to which button variant.

Reply:
Sorry, it looks like I wasted your time.

I thought that this was okay because we don't necessarily need to measure the bounce rate. All we want is to determine which button color gets clicked the most, which should be the case with this current implementation, unless I am missing something.

If we add some "page view" event (in `componentDidMount()` as you said), I still think we need to add a "click" event because it seems tricky to correlate events from different categories in GA.

Reply:
> If we add some "page view" event (in componentDidMount() as you said), I still think we need to add a "click" event because it seems tricky to correlate events from different categories in GA.

I'm pretty sure GA already has the concept of a user session so adding a "page view" with `variant=blue | green` still seems like it would work to me. I suggest asking Jorge how he plans to visualize it. I'd imagine he could make a report of all clicks (using our existing events and maybe summing together all categories) then filtering by which button variant the user was shown.

If you still think that we need a generic click event (i.e. without a specific category) then I'd rather see that added alongside the other existing event: https://github.com/mozilla/addons-frontend/blob/511d9666d402e4ba30f5d1790e04784967df8b70/src/core/installAddon.js#L436-L440

Reply:
So, last time, we setup the GA report ourselves. I can try to do the same with your approach though, that should be doable, but I'll be sure when I'll have enough data in GA -dev.

> If you still think that we need a generic click event (i.e. without a specific category) then I'd rather see that added alongside the other existing event: [/src/core/installAddon.js@`511d966`#L436-L440](https://github.com/mozilla/addons-frontend/blob/511d9666d402e4ba30f5d1790e04784967df8b70/src/core/installAddon.js#L436-L440)

We can't because we should also capture clicks when the mozAddonManager is not used.